diff --git a/build/defaultResolver.js b/build/defaultResolver.js
index 8f39ac5f04f24e38be9747aee41f17a56fee9f24..1725c8ff39daffc26af5cbeedcb38ac115af21e3 100644
--- a/build/defaultResolver.js
+++ b/build/defaultResolver.js
@@ -99,7 +99,7 @@ function _interopRequireWildcard(obj) {
  * LICENSE file in the root directory of this source tree.
  */
 function defaultResolver(path, options) {
-  if (process.versions.pnp && options.allowPnp !== false) {
+  if (process.versions.pnp === '1') {
     return (0, _jestPnpResolver().default)(path, options);
   }
 
